src/library/simplified/extension/currency-code.js

```diff
@@ -1,96 +1,6 @@
-}
-else if(utag.isFRateDetails() && b["entity.tripDetails.flightOffer.price.currency"] != undefined){
-    b["currencyCode"] = b["entity.tripDetails.flightOffer.price.currency"];
-}
-else if(utag.isFPymt()){
-    b["currencyCode"] = b["entity.checkout.cartTotal.currency"] ? b["entity.checkout.cartTotal.currency"] : "";
-}
-else if(utag.isFCO()){
-    b["currencyCode"] = b["entity.checkout.flightOffer.totalPrice.currency"] ? b["entity.checkout.flightOffer.totalPrice.currency"] : "";
-}
-
-//Car path
-else if(utag.isCarSR() && b["entity.carSearch.lowestOfferDailyPrice.currency"] != undefined){
-    b["currencyCode"] = b["entity.carSearch.lowestOfferDailyPrice.currency"];
-}
-else if (utag.isPCarSearch() && typeof b["entity.carSearch.lowestOfferTotalPrice.currency"] !== "undefined") {
-    b["currencyCode"] = b["entity.carSearch.lowestOfferTotalPrice.currency"];
-} 
-else if (utag.isPSR_HC() || utag.isPSR_FC() || utag.isPSR_FH_Responsive() || utag.isPIS()) {
-    if (typeof b.entity.packageSearch.results !== "undefined" && typeof b.entity.packageSearch.results.offers !== "undefined") {
-        b["currencyCode"] = b["entity.packageSearch.results.offers.0.packagePrice.packageTotalPrice.currency"] ? b["entity.packageSearch.results.offers.0.packagePrice.packageTotalPrice.currency"] : "";
-    } else if (typeof b["entity.multiItemSearch.multiItemSearchResults.offers.0.packagePrice.packageTotalPrice.currency"] !== "undefined") {
-        b["currencyCode"] = b["entity.multiItemSearch.multiItemSearchResults.offers.0.packagePrice.packageTotalPrice.currency"];
-    }
-}
-else if(utag.isCarSR() && b["entity.carSearch.lowestOfferDailyPrice.currency"] != undefined){
-    b["currencyCode"] = b["entity.carSearch.lowestOfferDailyPrice.currency"];
-}
-else if(utag.isCarUDP() && b["entity.carDetails.totalPrice.currency"] !== undefined) {
-	b["currencyCode"] = b["entity.carDetails.totalPrice.currency"];
-}
 
-//LX
-else if (utag.isLXS() && b["entity.activities.activitySearchResults.cheapestPrice.currency"] != undefined){
-    b["currencyCode"] = b["entity.activities.activitySearchResults.cheapestPrice.currency"];
-}
-
-//Rails Path
-else if (utag.isRailSearchResults() && b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
-    b["currencyCode"]  = b['entity.railSearch.searchResults.railLegs.0.railOfferItems.0.price.currency'];
-}
-
-else if (utag.isRailRateDetails() && b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
-    b['currencyCode'] = b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'];
-}
-
-//Common data element
-else if((utag.isCarPymt() || utag.isCarCO() || utag.isHCO() || utag.isHPymt()
-    || utag.isCruisePymt() || utag.isCruiseCO() || utag.isLXCO() || utag.isPCO() || utag.isPCO_Mobile() || utag.isMCO()) && typeof b["entity.checkout.cartTotal.currency"] != "undefined"){
```

Comment:
This might be the place to add your utag.isRCO

Reply:
yeah, this is actually the right place to add it.